src/TestData/wpf-test/WpfTest.cs

```diff
@@ -1,26 +1,32 @@
 ﻿// Copyright (c) Charlie Poole, Rob Prouse and Contributors. MIT License - see LICENSE.txt
 
 using System;
+using System.Windows;
 using System.Windows.Controls;
 using NUnit.Framework;
 
 // Test which resolves issue #1203
 namespace Test1
 {
     [TestFixture]
-    public class WPFTest
+    public class WPFTest : IWeakEventListener
```

Comment:
Is this just to force these tests to reference an assembly that causes it to be treated like a WPF app?

Reply:
NM, just read the original issue. :)